frontend/src/utils/format_column_data.ts

```
@@ -63,35 +57,121 @@ export class GraphData {
+        this.canvas.fillStyle = "lightseagreen";
+        this.canvas.beginPath();
+
+        const [x, y, _, children] = hovered_element;
+
+        this.canvas.arc(x, y, 10, 0, Math.PI * 2);
+        this.canvas.fill();
+
+        // Make the child connections all light up
+        for (const [l_x, l_y] of children) {
+          const cx = x,
+            cy = l_y;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)